posts/wiki/Writing Posts for this Wiki.md

```diff
@@ -1,96 +1,114 @@
 # Writing Posts for this Wiki
-## So you want to contribute to the Drauger OS Wiki. But you have no idea where to start. You've come to the right place!
-
+<center>
+## So you want to contribute to the Drauger OS Wiki. But you have no idea where to start.
```

Comment:
So you want to contribute to the Drauger OS wiki but you have no idea where to start.  You've come to the right place!